homeworks/03-data-structure/BinarySearchTree.ts

```diff
@@ -0,0 +1,53 @@
+import { IBinarySearchTree } from './IBinarySearchTree';
+import { IBinaryTree } from './IBinaryTree';
+
+export class BinarySearchTree implements IBinarySearchTree<number> {
```

Comment:
This class doesn't extend `BinaryTree` and, hence, doesn't have any method, e.g., `traverse`.